.github/workflows/lint.yml

```diff
@@ -10,24 +10,26 @@ jobs:
   build:
     name: lint
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        python-version: ["3.8", "3.11"]
     steps:
       - uses: actions/checkout@v3
-      - name: Set up Python ${{ matrix.python-version }}
+
+      - name: Set up Python
         uses: actions/setup-python@v3
         with:
-          python-version: ${{ matrix.python-version }}
-      - name: Install dependencies
+          python-version: 3.10.11
+
+      - name: Install python dependencies
         run: |
-          python -m pip install --upgrade pip
-          python -m pip install black~=22.0 pylint
           python -m pip install stochastic --ignore-requires-python
-          python -m pip install -r requirements-${{ matrix.python-version }}.txt
-      - name: Analyzing code with black
-        run: python -m black --check --diff .
-      - name: Analysing code with pylint
-        run: python -m pylint $(git ls-files '*.py' ':!:*apeworx*')
+          python -m pip install -r requirements.txt
+          ape plugins install .
+          python -m pip install .
+
       - name: Ensure clear Jupyter Notebooks
         uses: ResearchSoftwareActions/EnsureCleanNotebooksAction@1.1
+
+      - name: Run black
+        run: python -m black --check --diff .
+
+      - name: Run pylint
+        run: python -m pylint $(git ls-files '*.py')
```

Comment:
no, but I thought that was a goal to test ape

Reply:
oh unless we want to just exclude them from lint/pyright (maybe that's what you meant). but if we go in that direction, why not exclude the entire `examples` folder?

Reply:
there will be `apeworx` files that exist outside of `examples`. Presently, there is one:`utils/apeworx_integrations.py`. If you're able to get this to pass pylint & pyright without filling it with `ignore` comments, then lets include it in the CI. Otherwise, we should exclude it.

Reply:
leave all the apes alone! excluding them for now (until ape gets better typing)